server/server.js

```diff
@@ -142,24 +146,49 @@ Server.prototype._handleClientMessage = function(client, data) {
     return;
   }
 
-  logging.info('#client.message - received', client.id, message,
-     (this.resources[message.to] ? 'exists' : 'not instantiated'),
-     (this.subs[message.to] ? 'is subscribed' : 'not subscribed')
-    );
+  if (!this._clientDataPersist(client.id, message)) {
```

Comment:
This *client.id* is the old client - i.e. socket - yes, I agree it is confusing.  I will change that tomorrow.  In server/server.js, most of the references to *client* are to the old client (i.e. socket).